bionic/flow.py

```diff
@@ -1231,9 +1232,34 @@ def get(self, name, collection=None, fmt=None, mode=object):
             return set(values)
         elif collection is pd.Series or collection == "series":
             if len(result_group.key_space) > 0:
+                case_keys = [result.query.case_key for result in result_group]
+
+                resultgroups_by_entity_name = {
+                    entity_name: self._deriver.derive(
+                        entity_dnode_from_descriptor(entity_name)
+                    )
+                    for entity_name in result_group.key_space
+                }
+                values_by_entity_name_by_case_key = defaultdict(lambda: defaultdict())
+                for case_key in case_keys:
+                    for (
+                        entity_name,
+                        entity_result_group,
+                    ) in resultgroups_by_entity_name.items():
+                        for result in entity_result_group:
+                            result_case_key = result.query.case_key
+                            if (
+                                case_key.project(result_case_key.space)
+                                == result_case_key
+                            ):
+                                values_by_entity_name_by_case_key[case_key][
+                                    entity_name
+                                ] = result.value
+
                 index = multi_index_from_case_keys(
```

Comment:
At this point, the signature of this function is so complicated that I wonder if it even makes sense to keep it around rather than just inlining it.

Reply:
lol makes sense to inline it